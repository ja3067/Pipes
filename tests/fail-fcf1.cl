if True:
	def foo():
		return x
else:
	def bar():
		return x + 1

x = 3
print(foo())
print(bar())
