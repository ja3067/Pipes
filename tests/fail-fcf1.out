3
RuntimeError: name 'bar' is not defined
